sway-fmt-v2/src/utils/language/expr/code_block.rs

```diff
@@ -15,18 +15,26 @@ impl Format for CodeBlockContents {
         formatted_code: &mut FormattedCode,
         formatter: &mut Formatter,
     ) -> Result<(), FormatterError> {
-        writeln!(formatted_code)?;
-        for statement in self.statements.iter() {
-            statement.format(formatted_code, formatter)?;
-        }
-        if let Some(final_expr) = &self.final_expr_opt {
-            write!(
-                formatted_code,
-                "{}",
-                formatter.shape.indent.to_string(&formatter.config)?
-            )?;
-            final_expr.format(formatted_code, formatter)?;
+        if self.statements.is_empty() && self.final_expr_opt.is_none() {
```

Comment:
could be written to just be an if statement without the else

Reply:
I.e.
```rust
if !self.statements.is_empty() || self.final_expr_opt.is_some() {
```

Reply:
Thanks (: